includes/admin/class-wc-stripe-subscription-detached-bulk-action.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WC_Stripe_Subscription_Detached_Bulk_Action
+ */
+class WC_Stripe_Subscription_Detached_Bulk_Action {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		if ( WC_Stripe_Woo_Compat_Utils::is_custom_orders_table_enabled() ) {
+			add_filter( 'bulk_actions-woocommerce_page_wc-orders--shop_subscription', [ $this, 'subscriptions_bulk_actions' ] );
+			add_filter( 'handle_bulk_actions-woocommerce_page_wc-orders--shop_subscription', [ $this, 'handle_subscription_detachment_check' ], 10, 3 );
+		} else {
+			add_filter( 'bulk_actions-edit-shop_subscription', [ $this, 'subscriptions_bulk_actions' ] );
+			add_filter( 'handle_bulk_actions-edit-shop_subscription', [ $this, 'handle_subscription_detachment_check' ], 10, 3 );
+		}
+	}
+
+	/**
+	 * Add custom bulk action to check for detachment of subscriptions' payment methods.
+	 *
+	 * @param $bulk_actions array An associative array of actions which can be performed on the subscription post type.
+	 * @return array
+	 */
+	public function subscriptions_bulk_actions( $bulk_actions ) {
+		$bulk_actions['check-for-payment-method-detachment'] = __( 'Check for payment method detachment', 'woocommerce-gateway-stripe' );
```

Comment:
This action name is perfectly aligned with the feature. But I want to suggest another alternative. We can also call it `Payment method health`.  This optional and just my preference.